packages/dds/merge-tree/src/properties.ts

```diff
@@ -168,16 +168,5 @@ export function extendIfUndefined<T>(base: MapLike<T>, extension: MapLike<T> | u
 
 // Create a MapLike with good performance.
 export function createMap<T>(): MapLike<T> {
-    const map = Object.create(null);
-
-    // Using 'delete' on an object causes V8 to put the object in dictionary mode.
-    // This disables creation of hidden classes, which are expensive when an object is
-    // constantly changing shape.
-    // eslint-disable-next-line @typescript-eslint/dot-notation
-    map["__"] = undefined;
-    // eslint-disable-next-line @typescript-eslint/dot-notation, @typescript-eslint/no-dynamic-delete
-    delete map["__"];
-
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-return
-    return map;
+    return Object.create(null) as MapLike<T>;
```

Comment:
question, is it better to use `Object.create` over `{}` for any reason?

Reply:
Yeah. I'm told that in V8 Object.create(null) actually has the effect that the delete was trying to accomplish. It's taken as a signal to use a non-shared map for the object.